docs/behat/basic-usage.rst

```diff
@@ -0,0 +1,513 @@
+Basic Usage
+===========
+
+In our opinion the best way to understand how some stuff works is by showing some examples so this section gather all knowledge form previous chapters.
+Assume that we are implementing managing countries in our system.
+Let the fun begin.
+
+Describing feature
+------------------
+First of all let's start with writing our feature file the most crucial thing here is ubiquitous language. We don't want to specify how something is done,
+it doesn't really matter in case of business.
```

Comment:
First of all let's start with writing our feature file - the most crucial thing here is the ubiquitous language, the main principle of BDD. We have to describe the functionalities of our system in a way all the project stakeholders will understand, by using words and sentences that sound natural and will in the end form a user story.